src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/Components/ComponentMarkupEncodingPassTest.cs

```diff
@@ -95,12 +95,12 @@ public void Execute_MixedHtmlContent_NewLine_SetsEncoded()
         }
 
         [Fact]
-        public void Execute_MixedHtmlContent_Ampersand_SetsEncoded()
+        public void Execute_MixedHtmlContent_Ampersand_DoesNotSetEncoded()
```

Comment:
When there is a '&' that is not part of an HTML entity, should we call `AddMarkupContent` or `AddContent`? Before this change, we call `AddMarkupContent` but I changed that behavior. Is that wrong?

Reply:
Seems right to me.

Reply:
I think it would be good to let things be errors that are errors in html. 

Like if you type `Ted Cruz &amp; Tom Cruz` in a static HTML file - does it work? (yes)
So what happens if you type `Ted Cruz & Tom Cruz` in a static HTML file?

Let's try to apply a consitent set of rules and not guess about user intent.

Reply:
If we need to flag this as an error we should.